lib/logstash/plugin_mixins/elasticsearch/common.rb

```diff
@@ -204,8 +204,15 @@ def submit(actions)
         return
       end
 
+      responses = bulk_response["items"]
+      if responses.size != actions.size # can not map action -> response reliably
```

Comment:
could be more explicit in the comment saying 
"to deal with weird behavior on ES 7.10.2, 7.11.1 where a _bulk request to
index X documents returned X+Y entries"